src/queryparser.py

```diff
@@ -0,0 +1,16 @@
+
+def parse(filter: str, varList: list):
+
+    len_varList = len(varList)
+    n = len_varList
+    adaptetd_var_list = []
```

Comment:
Typo I assume "adaptetd" should be "adapted".

Reply:
Thank you, it was fixed